globus_automate_client/flows_client.py

```diff
@@ -205,14 +229,14 @@ def deploy_flow(
         title: str,
         subtitle: Optional[str] = None,
         description: Optional[str] = None,
-        keywords: List[str] = [],
-        visible_to: List[str] = [],
-        runnable_by: List[str] = [],
-        administered_by: List[str] = [],
+        keywords: Iterable[str] = (),
+        flow_viewers: Iterable[str] = (),
+        flow_starters: Iterable[str] = (),
+        flow_administrators: Iterable[str] = (),
         subscription_id: Optional[str] = None,
         input_schema: Optional[Mapping[str, Any]] = None,
         validate_definition: bool = True,
-        validate_input_schema: bool = True,
+        validate_schema: bool = True,
```

Comment:
I think we had this JSONSchema validation opt-out-able because we were trying to get a feel for how it was working. It's been a few months and no ones complained. I'm open to removing these flags altogether and making validation an SDK staple (:

Reply:
I think I'm good with continuing to make opt-in for validation the default, but allow people to override if they really want to. Can I imagine a situation where validation would blow up on the client but then succeed on the server? Especially in the case of the schema I cannot since I believe we're using the exact same validation from `jsonschema`, but I guess it feels like one of those things where I fear someone will complain that the client is limiting them somehow. I dunno...